tests/functional/pv/pv_services/test_cephfs_fsync_consistency.py

```diff
@@ -0,0 +1,93 @@
+import logging
+
+from ocs_ci.framework.testlib import ManageTest, tier2, bugzilla, green_squad
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import get_worker_nodes
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2212310")
+@green_squad
+class TestCephfsFsyncConsistency(ManageTest):
+    """
+    Ensuring File Completeness Post-fsync with Shared CephFS Volume
+
+    """
+
+    def test_cephfs_fsync_consistency(self, teardown_project_factory):
+        """
+        Procedure:
+            1.Set Up Shared Storage: Create a PVC with sc=cephfs and mode=RWX.
+            2.Deploy Client Pod on Worker Node A, attaching the shared PVC.
+            3.Deploy Server Pod on Worker Node B, attaching the same PVC.
+            4.Client Pod write a file to the shared PVC, then call fsync to ensure data is flushed to disk.
+            5.Server Pod Read the file
+        """
+        project_name = "cephfs-rwx-ns"
+        pod_name = "cephfs-rwx-pod"
+        project_obj = helpers.create_project(project_name=project_name)
+        worker_node_names = get_worker_nodes()
+        teardown_project_factory(project_obj)
+        logger.info(
+            f"Created new pvc sc_name={constants.CEPHFILESYSTEM_SC} namespace={project_name}, "
+            f"size=6Gi, access_mode={constants.ACCESS_MODE_RWO}"
+        )
+        pvc_obj = helpers.create_pvc(
+            sc_name=constants.CEPHFILESYSTEM_SC,
+            namespace=project_name,
+            size="6Gi",
+            do_reload=False,
+            access_mode=constants.ACCESS_MODE_RWX,
+        )
+        logger.info(
+            f"Create new pod. Pod name={pod_name},"
+            f"interface_type={constants.CEPHBLOCKPOOL}"
+        )
+        pod_obj_client = helpers.create_pod(
+            pvc_name=pvc_obj.name,
+            namespace=project_obj.namespace,
+            interface_type=constants.CEPHBLOCKPOOL,
```

Comment:
The PVC is created with constants.CEPHFILESYSTEM_SC, which refers to CephFS, but the pods are being created with constants.CEPHBLOCKPOOL as the interface_type, does it works?

Reply:
fixed 
```
$ oc get pvc -n cephfs-rwx-ns
NAME                                       STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS                VOLUMEATTRIBUTESCLASS   AGE
pvc-test-6c5c86da89904beeb412eecd9502073   Bound    pvc-7a67273a-7977-470a-b60b-36604ecb48cb   6Gi        RWX            ocs-storagecluster-cephfs   <unset>                 94s


$ oc get pods client -o yaml | grep -i pvc-test-6c5c86da89904beeb412eecd9502073 -A 1 -B 3
  volumes:
  - name: mypvc
    persistentVolumeClaim:
      claimName: pvc-test-6c5c86da89904beeb412eecd9502073

$ oc get pods server -o yaml | grep -i pvc-test-6c5c86da89904beeb412eecd9502073 -A 1 -B 3
  volumes:
  - name: mypvc
    persistentVolumeClaim:
      claimName: pvc-test-6c5c86da89904beeb412eecd9502073

```